### ISSUE TITLE
8.1.2 new style of product page has a critical problem on short description if not present the p tag

### ISSUE THREAD

**Issue by Maofree** [PrestaShop/PrestaShop#34347]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Hi
I've found a big problem on the new style of the product pages - admin side

it is present only on the short description

if in the ps_product_lang there is not the <p> </p> tags to limit the short descriptions there is a javascript error which blocks the product page without giving the possibility to make changes

![5](https://github.com/PrestaShop/PrestaShop/assets/653230/45fed7c4-a8ed-4182-8bc3-90a35d6e7206)
![6](https://github.com/PrestaShop/PrestaShop/assets/653230/06ed3a4c-a695-4e3a-9f70-a4021fd2331f)

I got this error from a site of my customer that use csv import. so in the file there are no present html code and <p> to open and </p> to close thje descriptions

this problem is not present in the long description, if I set from phpmyadmin a text without p tags I can save from the product page and the p tags will be creates subsequently
the same should be done for the short description

is it possible to fix this problem on the next 8.1.3? I think is important and many sites use external imports

### Steps to reproduce

1. take a product to do the test and active the new style
2. go to phpmyadmin into ps_product_lang
3. remove <p> and </p> from the short description of a language of this product
4. go to the product page, admin side
5. try to do something and open the console log

### PrestaShop version(s) where the bug happened

8.1.2

### PHP version(s) where the bug happened

8.1

**Comment by AureRita** [comment]
Hi @maofree 

Thank you for your report, I tried to reproduce your issue as you can see : 

[Untitled_ Oct 24, 2023 11_46 AM.webm](https://github.com/PrestaShop/PrestaShop/assets/100699445/e84c64df-58ca-4e7a-a87a-af473f619aba)

as you can see at the start I don't reproduce the issue, but after other test, I find the issue. 

It's because you put a dot (.) on your short description as you can see :

[Untitled_ Oct 24, 2023 11_59 AM.webm](https://github.com/PrestaShop/PrestaShop/assets/100699445/5d1b9f02-8cc4-4ced-91ea-2ade01732a6a)

Because I reproduce the issue with PrestaShop version 8.1.2, I'll add this to the backlog so it can be fixed.

Please be aware that some issues might take a very long time to be resolved.

Thanks

**Comment by Hlavtox** [comment]
@PrestaShop/prestashop-core-developers 

Happens only only new product page, tinymce fields in other pages are working fine.

Probably something to do with "watching for changes". Is it even needed? Why we can't enable the button right away?

**Comment by RosaBenouamer** [comment]
Is the problem only on dot ?

**Comment by Hlavtox** [comment]
@RosaBenouamer on all texts that do not start/contain html tags

**Comment by M0rgan01** [comment]
PR: https://github.com/PrestaShop/PrestaShop/pull/34600

**Comment by Jenny-Glamorhair** [comment]
Ive done a fresh new install and this problem came up after 2 days. it was first working but stopped this morning. I found that contents added by csv import. its also not working with Description. the solution is to remove the content, paste into notepad to remove all tags, paste back and save. it works. for by system, the P tags wasnt the problem as I pasted in without tags and it saved. it just not working with csv tags. I had even typed out the tags and it didint work.
